README.md

```diff
@@ -164,6 +164,14 @@ The following environment variables configure Livebook:
   * LIVEBOOK_APP_SERVICE_URL - sets the application url to manage this
     Livebook instance within the cloud provider platform.
 
+  * LIVEBOOK_APPS_PATH - the directory with app notebooks. When set, the apps
+    are deployed on Livebook startup with the persisted settings.
+    Password-protected notebooks will receive a random password,
+    unless `LIVEBOOK_APPS_PATH_PASSWORD` is set.
+
+  * LIVEBOOK_APPS_PATH_PASSWORD - the password to use for all protected apps
+    deployed from `LIVEBOOK_APPS_PATH`.
```

Comment:
```suggestion
    unless LIVEBOOK_APPS_PATH_PASSWORD is set.

  * LIVEBOOK_APPS_PATH_PASSWORD - the password to use for all protected apps
    deployed from LIVEBOOK_APPS_PATH.
```

My bad. We don't use markdown because it is also rendered on the CLI.